app/Http/Controllers/FlightGetController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Flight;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class FlightGetController
+{
+    public function __invoke(Request $request): JsonResponse
+    {
+        $query = Flight::with('origin', 'destination', 'airline');
+
+        $sort = $request->query('sort');
+        $sortOrder = $request->query('sortOrder');
+        $startDate = $request->query('start_date');
+        $endDate = $request->query('end_date');
+
+        $query->when($sort && in_array($sortOrder, ['asc', 'desc']), function ($query) use ($sort, $sortOrder) {
```

Comment:
`in_array($sortOrder, ['asc', 'desc']` should be handled in the validation, seems like we missed the FormRequest class here! 
FormRequest apply for get requests as well, not just things with body, they validate queries as well